Tests/Recurly/Base_Test.php

```diff
@@ -63,4 +63,27 @@ public function testCheckIfResponseContainGetHeadersFunction() {
     $this->assertTrue(method_exists($adjustment, 'getHeaders'),'Adjustments Class does not have method getHeaders');
     $this->assertInternalType('array', $adjustment->getHeaders());
   }
+
+  public function testPassingEmptyResourceCode() {
+    $this->client->addResponse('GET', '/accounts/abcdef1234567890', 'accounts/show-200.xml');
+    try {
+      $uri = Recurly_Base::_safeUri(Recurly_Client::PATH_ACCOUNTS, "");
```

Comment:
The exception is getting thrown here right? so why do we need the next 2 lines of code?

Reply:
I revised this but what's the reason for now using `$this->fail`?

Reply:
Nevermind, I utilized `expectException`.